src/execution/execute.ts

```diff
@@ -976,7 +988,7 @@ function handleFieldError(
 
   // If the field type is non-nullable, then it is resolved without any
   // protection from errors, however it still properly locates the error.
-  if (isNonNullType(returnType)) {
+  if (exeContext.propagateErrors && isNonNullType(returnType)) {
```

Comment:
This is almost exactly the same as I had in #4192 except you called it propagateErrors whereas I called it `errorPropagation` :laughing: 

https://github.com/graphql/graphql-js/blob/eb9b6c8a77193fc5ed3685d1e95d32b99c9df9d9/src/execution/execute.ts#L609

(I don't have a particular preference between the terms.)

Reply:
I renamed it to `errorPropagation`